deploy/nl/embeddings.yaml

```diff
@@ -35,36 +35,12 @@ indexes:
     store: LANCEDB
     embeddings: gs://datcom-nl-models-dev/lancedb/lancedb_embeddings_medium_2024_04_09_07_22_21.ft_final_v20230717230459.all-MiniLM-L6-v2
     model: ft-final-v20230717230459-all-MiniLM-L6-v2
-  medium_vertex_ft:
-    store: VERTEXAI
-    project_id: datcom-website-dev
-    location: us-central1
-    index_endpoint_root: 302175072.us-central1-496370955550.vdb.vertexai.goog
-    index_endpoint: projects/496370955550/locations/us-central1/indexEndpoints/8500794985312944128
-    index_id: dc_all_minilm_l6_v2_ft_1709655496660
-    model: dc-all-minilm-l6-v2-model
-  medium_vertex_uae:
-    store: VERTEXAI
-    project_id: datcom-website-dev
-    location: us-central1
-    index_endpoint_root: 302175072.us-central1-496370955550.vdb.vertexai.goog
-    index_endpoint: projects/496370955550/locations/us-central1/indexEndpoints/8500794985312944128
-    index_id: uae_large_v1_1709655438364
+  medium_uae:
+    store: MEMORY
+    embeddings: gs://datcom-nl-models/embeddings_medium_2024_05_13_22_58_40.4910007712298827776.csv
```

Comment:
TODO: we probably want to add the model name / index name in full to the file name?

Reply:
ok, we should have a naming convention..